__tests__/ExtendedString.test.js

```diff
@@ -0,0 +1,172 @@
+require("../StringPrototypeExtension");
```

Comment:
Please rename this test file to be in line with the name of the implementation file in Javascript. Also, rename the test folder from __tests__ to tests.